### ISSUE TITLE
[Feature Reqeust] Ability to set transaction fees when closing a channel?

### ISSUE THREAD

**Issue by ParmuSingh** [ACINQ/eclair-mobile#184]
Currently it reserves 22,000 sats within channel to spend on tx fees when closing channel. That is quite fairly high and a lot of time people can wait more blocks for their funds to come onchain. A feature to set tx fees for closing channel seems like a good idea. It'll also help reduce extra channel reserve.

**Comment by brianddk** [comment]
Yes please.  I see the point of LN as a way to let users background process mainnet TXNs for hours or days.  I almost always set my mainnet TXNS for 12h.  I'm fine waiting a week or so for a channel open or close, though it rarely takes more than a good nights sleep.
